test/e2e/sgx/sgx.go

```diff
@@ -43,42 +41,46 @@ func init() {
 func describe() {
 	f := framework.NewDefaultFramework("sgxplugin")
 
-	kustomizationPath, err := utils.LocateRepoFile(kustomizationYaml)
-	if err != nil {
-		framework.Failf("unable to locate %q: %v", kustomizationYaml, err)
-	}
-
-	ginkgo.It("checks availability of SGX resources", func() {
-		tmpDir, err := os.MkdirTemp("", "sgxplugine2etest-"+f.Namespace.Name)
+	ginkgo.BeforeEach(func() {
+		kustomizationPath, err := utils.LocateRepoFile(kustomizationYaml)
 		if err != nil {
-			framework.Failf("unable to create temp directory: %v", err)
+			framework.Failf("unable to locate %q: %v", kustomizationYaml, err)
 		}
-		defer os.RemoveAll(tmpDir)
+		_ = utils.DeployWebhook(f, kustomizationPath)
 
-		err = utils.CreateKustomizationOverlay(f.Namespace.Name, filepath.Dir(kustomizationPath), tmpDir)
-		if err != nil {
-			framework.Failf("unable to create kustomization overlay: %v", err)
+		msg := framework.RunKubectlOrDie("node-feature-discovery", "apply", "-k", "../../deployments/sgx_nfd")
```

Comment:
Is it safe to use a relative path to `kustomization.yaml` here given the test can be started outside of the repo's root? Probably using `utils.LocateRepoFile()` would be safer.

Reply:
fixed. we're resolving the repo path quite many times. might make sense to move to Suite init at some point?

Reply:
Agree, this might be useful